src/zns/utils.ts

```diff
@@ -28,7 +34,7 @@ function isAddress(address: string) {
  * For example, converts byte buffers to buffers of maximum 5 bit numbers,
  * padding those numbers as necessary. Necessary for encoding Ethereum-style
  * addresses as bech32 ones.
- *
+ * @internal 
```

Comment:
@JohnnyJumper Why this is internal if we don't import this file for typedoc? Here and below.

Reply:
This is probably left from the old setup, I will remove the unnecessary ones